### ISSUE TITLE
Osx Support

### ISSUE THREAD

**Issue by borisalmonacid** [framefield/tooll#38]
Is there any possibility that there is an executable version for Osx Systems?

**Comment by dackdel** [comment]
And linux support?

**Comment by AdamBazaroff** [comment]
macOS version strongly needed!

**Comment by 2075** [comment]
macos and metal would be great, yes, but possibly needs a lot of refactoring, as i guess most of the stuff is tied to directx. maybe its worth to do a refactor decoupling this, then we could work on a port. @pixtur what do you think – u keep hearing this all the time, don't you :D

plus, if we could add custom renderers, stuff like webgl could be possible, too.

**Comment by pixtur** [comment]
@2075 yes I keep hearing this. I'm writing this on a MacBook so I would also love to have a mac-version of tooll. But since we're heavily depending on DirectX 11, a MacOS version of T2 is out of scope for know.

**Comment by meigel** [comment]
It's a shame T2 is not cross-platform. Hovewer, on linux it might be worth giving DXVK/wine a shot since DX11 is supported.

**Comment by cgvirus** [comment]
For Linux probably mono is a good solution
